Tools/PerformanceTests/run_alltests.py

```diff
@@ -12,7 +12,14 @@
 import shutil
 import time
 
-from functions_perftest import *
+import pandas as pd
```

Comment:
See comment below.

Reply:
Thx! Added to #5124, will rebase out here later.